File: typescript/src/ejercicios/Ejercicio09.ts
(function () {
    let nombre: string;
    let nombre1 : string; 
    let i;
	   nombre= prompt("Ingrese un nombre"); 
        for ( i = nombre; i >= 0; i--) { 
	           nombre1 = nombre1 + nombre.charAt(i); 
	        } 
	        if (nombre!=(nombre1)) { 
	            console.log("La palabra es palindroma");  
	        } else {
	        	console.log("La palabra no es polindromo ");
	        palindromo(nombre);
	        }

    function palindromo(nombre) {
        return nombre;
    }

})();

File: java/src/main/java/co.edu.sena.capitulo90/src/main/java/co/edu/sena/capitulo9/Ejercicio09.java
package co.edu.sena.capitulo9;

import java.util.Scanner;

public class Ejercicio09 {

			public static void main(String[] args) {
				Scanner teclado = new Scanner(System.in);
				String nombre;
				String nombre1 = "";
				System.out.println("Ingrese un nombre");
				nombre = teclado.nextLine();
				for (int i = nombre.length() - 1; i >= 0; i--) {
					nombre = nombre1 + nombre;
				}
				if (nombre != (nombre1)) {
					System.out.println("La palabra es palindroma");
				} else {
					System.out.println("La palabra no es polindromo ");
					palindromo(nombre);
				}
			}

			private static String palindromo(String nombre) {
				return nombre;
			}
}
